src/db/models/User.ts

```diff
@@ -210,7 +210,17 @@ UserModel.init(
             defaultValue: {},
         },
     },
-    { sequelize: sequelizeConnection, modelName: 'users', timestamps: false },
+    {
+        sequelize: sequelizeConnection,
+        modelName: 'users',
+        timestamps: false,
+        hooks: {
+            beforeValidate: (instance) => {
```

Comment:
this transforms `''` to `null` before model validation
then isEmail doesn't fail for `''`
also database receives `null` instead `''`